src/ai/backend/manager/registry.py

```diff
@@ -1242,11 +1224,31 @@ async def _post_enqueue() -> None:
                             else:
                                 matched_dependency_session_ids.append(match_info.id)
 
+                    if sudo_session_enabled:
+                        environ["SUDO_SESSION_ENABLED"] = "1"
+
+                    session_data["environ"] = environ
+                    session_data["requested_slots"] = session_requested_slots
+                    session = SessionRow(**session_data)
+                    kernels = [KernelRow(**kernel) for kernel in kernel_data]
+                    db_sess.add(session)
+                    db_sess.add_all(kernels)
```

Comment:
```suggestion
                    db_sess.add_all(kernels)
                    await db_sess.flush()
```
This will help avoiding potential `IntegrityError`: https://docs.sqlalchemy.org/en/20/orm/session_transaction.html#managing-transactions

Reply:
thank you! I checked it and it now works